packages/core/src/service/services/asset.service.ts

```diff
@@ -565,8 +568,8 @@ export class AssetService {
             case 'Collection':
                 return 'collectionId';
             default:
-                throw new InternalServerError('error.could-not-find-matching-orderable-asset');
-        }
+                return `${camelCase(entityName, true)}Id`;
```

Comment:
@michaelbromley I accidentally added true flag in this function, can you please remove true flag that will actually turn the first letter capital making this Title case instead of camelCase, 

sorry for the inconvenience, thanks!

Reply:
Done in v1.1.5